src/editortoolkit.cpp

```diff
@@ -625,6 +639,139 @@ std::string EditorToolkit::EditInfo()
     return m_editInfo;
 }
 
+bool EditorToolkit::Group(std::string groupType, std::vector<std::string> elementIds)
+{
+    Object *newParent, *neumeParent, *newSylParent, *sylParent;
+    std::set<Object *> parents;
+
+    if (!m_doc->GetDrawingPage()) {
+        LogError("Could not get the drawing page.");
+        return false;
+    }
+
+    if(groupType.compare("nc") == 0){
```

Comment:
`groupType` is a string so it's safe to use `==` to compare with other strings. For `std::string` it seems `compare` is used more for sorting.